spec/cli/navigate_spec.rb

```diff
@@ -54,6 +54,38 @@ def require_relative(path)
     end
   end
 
+  context "finds the right API", :vcr do
+    def stub_with_invalid
+      stub_const('Razor::CLI::Navigate::PRIORITY_RAZOR_API', 'https://bad-priority:8151/api')
+      stub_const('Razor::CLI::Navigate::DEFAULT_RAZOR_API', 'https://bad-default:8151/api')
+    end
+    let(:cli_string) do
+      %w[--version].freeze
+    end
+    it "chooses the -u argument first" do
+      cli_string = %w[-u http://localhost:8150/api --version].freeze
+      nav = Razor::CLI::Parse.new(cli_string).navigate
+      nav.last_url.to_s.should == 'http://localhost:8150/api'
```

Comment:
Could we set these tests urls to something totally different to make sure this does what we think it does? I'm imagining that this would come out the same if it was using defaults.

Reply:
Updated.